src/js/components/Button/__tests__/Button-test.js

```diff
@@ -205,6 +205,30 @@ describe('Button', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  test('normalized', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Button onClick={() => {}} normalized>
+          normalized
+        </Button>
+      </Grommet>,
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('normalized primary', () => {
```

Comment:
should we also add test for icon w/o a label?

Reply:
Yes, we should. Good call.